apps/examples/connect_test/connect_test_client.c

```diff
@@ -0,0 +1,676 @@
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License\n");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <tinyara/config.h>
+#include <stdio.h>
+#include <errno.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <sys/socket.h>
+#include <sys/time.h>
+#include <sys/stat.h>
+#include <wifi_manager/wifi_manager.h>
+#include <net/if.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include "connect_test.h"
+#include "connect_test_log.h"
+#include "connect_test_utils.h"
+
+#define TAG "[CT_CLIENT]"
+#define WT_BUF_SIZE 1024
+#define WT_ACK_SIZE 128
+#define WT_UDP_SERVER_ADDR CONFIG_LWIP_DHCPS_SERVER_IP
+#define WT_UDP_PORT 8888
```

Comment:
UDP and TCP can use same port. If I'm right, you don't need to differentiate the port number.

Reply:
Changes done as suggested